python/dgl/graphbolt/utils/sample_utils.py

```diff
@@ -228,27 +291,39 @@ def unique_and_compact_node_pairs(
         dst_nodes[dst_type].append(dst_node)
     src_nodes = {ntype: torch.cat(nodes) for ntype, nodes in src_nodes.items()}
     dst_nodes = {ntype: torch.cat(nodes) for ntype, nodes in dst_nodes.items()}
-    # Compute unique destination nodes if not provided.
-    if unique_dst_nodes is None:
-        unique_dst_nodes = {
-            ntype: torch.unique(nodes) for ntype, nodes in dst_nodes.items()
-        }
+    # Compute destination nodes if not provided.
+    if seeds is None:
+        if deduplicate:
+            seeds = {
+                ntype: torch.unique(nodes) for ntype, nodes in dst_nodes.items()
+            }
+        else:
+            seeds = dst_nodes
 
     ntypes = set(dst_nodes.keys()) | set(src_nodes.keys())
-    unique_nodes = {}
+    cat_seeds = {}
```

Comment:
What's cat_seed? cat?

Reply:
The seeds after torch.cat. Maybe can changed into new_seeds?